types/src/block_metadata.rs

```diff
@@ -31,7 +31,7 @@ pub struct BlockMetadata {
     epoch: u64,
     round: u64,
     previous_block_votes: Vec<bool>,
-    proposer: u64,
+    proposer: Option<u64>,
```

Comment:
since we're editing this, we'd also need to change their openapi.yaml and regenerate the spec.html - this then trickles into re-generating the typescript SDK types